presto-hive/src/main/java/com/facebook/presto/hive/HiveUtil.java

```diff
@@ -514,6 +515,11 @@ public static boolean isPrestoView(Table table)
         return "true".equals(table.getParameters().get(PRESTO_VIEW_FLAG));
     }
 
+    public static boolean isHiveView(Table table)
+    {
+        return table.getTableType().equals(TableType.VIRTUAL_VIEW.name());
```

Comment:
For Presto view, the table type is also VIRTUAL_VIEW.

Reply:
@jzhuge sorry missed checking this message ... are you interested in this patch ? I can rebase and fix this if you require it.

Reply:
That'd great. Is there a JIRA number for this PR?

More comments to follow.

Reply:
@jzhuge This is the issue: https://github.com/prestodb/presto/issues/7338 ... I will try to rebase it and fix your comment.